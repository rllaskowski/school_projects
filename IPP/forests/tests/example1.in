ADD Las_Palmas
ADD Las_Vegas
ADD Czarnolas buk sikorka
ADD Czarnolas brzoza
ADD Czarnolas
ADD Czarnolas buk pierzaczek
ADD Las_Palmas spruce wasp
ADD Las_Palmas spruce wasp
ADD Las_Palmas fir wasp
ADD Las_Palmas spruce owl
PRINT
CHECK Las_Kabacki
CHECK * * wasp
CHECK Las_Vegas * pierzaczek
DEL Czarnolas buk sikorka
DEL Las_Palmas
PRINT Las_Palmas
CHECK Czarnolas buk pierzaczek
PRINT Czarnolas buk
