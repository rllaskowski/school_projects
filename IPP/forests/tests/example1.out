OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
Czarnolas
Las_Palmas
Las_Vegas
NO
YES
NO
OK
OK
YES
pierzaczek
